system/bluetooth1/device.go

```diff
@@ -229,6 +233,7 @@ func newDevice(systemSigLoop *dbusutil.SignalLoop, dpath dbus.ObjectPath) (d *de
 	d.Icon, _ = d.core.Icon().Get(0)
 	d.RSSI, _ = d.core.RSSI().Get(0)
 	d.blocked, _ = d.core.Blocked().Get(0)
+	d.Battery, _ = d.core.Percentage().Get(0)
```

Comment:
Percentage is not found, and line 500 is same
![image](https://github.com/linuxdeepin/dde-daemon/assets/79958313/78dff06a-14ed-41cb-af3e-ff92c9f55eaf)

Reply:
this PR need https://github.com/linuxdeepin/go-dbus-factory/pull/100

Reply:
ok, https://github.com/linuxdeepin/go-dbus-factory/pull/100 have been merged